Gemfile

```diff
@@ -18,6 +18,7 @@ gem 'lograge'
 gem 'logstash-event'
 gem 'paper_trail'
 gem 'pg', '>= 0.18', '< 2.0'
+gem 'pg_trgm'
```

Comment:
I don't think this gem is being used, just the pg extension via the Textacular gem.

Reply:
Admittedly, I am not entirely sure how this gem is used. It is recommended in the [Textacular documentation](https://github.com/textacular/textacular#usage) to use with fuzzy search